docs/source/getting-started/installing.adoc

```diff
@@ -42,8 +42,7 @@ Windows::
 +
 [NOTE]
 ====
-To use {project} with Hyper-V ensure that, after you link:https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/quick-start/enable-hyper-v[install Hyper-V], you also link:https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/quick-start/connect-to-network[add a Virtual Switch] using the Hyper-V Manager.
-For specific configuration steps see the xref:../getting-started/setting-up-driver-plugin.adoc#[Setting Up the Driver Plug-in] section.
+To use {project} with Hyper-V ensure that, after you link:https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/quick-start/enable-hyper-v[install Hyper-V], you also link:https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/quick-start/connect-to-network[add a Virtual Switch] by setting the environment variable `HYPERV_VIRTUAL_SWITCH` to the name of the external virtual switch. For specific configuration steps see the xref:../getting-started/setting-up-driver-plugin.adoc#[Setting Up the Driver Plug-in] section.
```

Comment:
To use {project} with Hyper-V ensure that, after you link:https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/quick-start/enable-hyper-v[install Hyper-V], link:https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/quick-start/connect-to-network[add a Virtual Switch] and set the environment variable `HYPERV_VIRTUAL_SWITCH` to this virtual switch. or specific configuration steps see the xref:../getting-started/setting-up-driver-plugin.adoc#[Setting Up the Driver Plug-in] section.